client/src/components/KeyConceptList/KeyConceptList.tsx

```diff
@@ -6,7 +6,7 @@ import "./KeyConceptList.scss";
 
 interface KeyConceptListProps {
   question_answer_pairs: [React.ReactNode, React.ReactNode][];
-  compact: boolean;
+  compact?: boolean;
```

Comment:
before I made `compact` optional, I got an error in `StandardFAQ.tsx` telling me that `KeyConceptList` was missing `compact`

Any ideas for a better approach appreciated